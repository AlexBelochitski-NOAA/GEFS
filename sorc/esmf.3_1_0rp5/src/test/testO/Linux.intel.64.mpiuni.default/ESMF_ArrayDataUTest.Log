20190610 112238.204827 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112238.205099 INFO     PET0  Beginning Test, file ESMF_ArrayDataUTest.F90, line          65
20190610 112238.205117 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112238.205451 INFO     PET0  PASS  Creating an Array from allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line          89
20190610 112238.205579 INFO     PET0  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line          97
20190610 112238.207744 INFO     PET0  PASS  Obtaining access to data in Array via F90 array pointer, ESMF_ArrayDataUTest.F90, line         105
20190610 112238.207780 INFO     PET0  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTest.F90, line         118
20190610 112238.207797 INFO     PET0  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         133
20190610 112238.207829 INFO     PET0  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTest.F90, line         146
20190610 112238.207860 INFO     PET0  PASS  Destroying Array created from an allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         155
20190610 112238.208005 INFO     PET0  PASS  Creating an Array from allocated F90 array pointer using ESMF_DATA_COPY, ESMF_ArrayDataUTest.F90, line         164
20190610 112238.208024 INFO     PET0  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         172
20190610 112238.208042 INFO     PET0  PASS  Obtaining access to data in Array via F90 array pointer, ESMF_ArrayDataUTest.F90, line         180
20190610 112238.208074 INFO     PET0  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTest.F90, line         193
20190610 112238.208089 INFO     PET0  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         208
20190610 112238.208130 INFO     PET0  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTest.F90, line         221
20190610 112238.208148 ERROR    PET0 ESMF_LocalArrayCreate.F90 22588 ESMF_LocalArrayF90Deallocate Unable to allocate requested memory LocalArray Deallocation
20190610 112238.208161 INFO     PET0  PASS  Destroying Array created from an allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         230
20190610 112238.208183 INFO     PET0  PASS  Creating an Array from allocated F90 array pointer slice, ESMF_ArrayDataUTest.F90, line         242
20190610 112238.208199 INFO     PET0  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         250
20190610 112238.208234 INFO     PET0  PASS  Obtaining access to data in Array via F90 array pointer, ESMF_ArrayDataUTest.F90, line         258
20190610 112238.208266 INFO     PET0  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTest.F90, line         271
20190610 112238.208281 INFO     PET0  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         286
20190610 112238.208324 INFO     PET0  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTest.F90, line         299
20190610 112238.208336 INFO     PET0  PASS  Destroying Array created from an allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         308
20190610 112238.208357 INFO     PET0  PASS  Creating an Array from allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         325
20190610 112238.208373 INFO     PET0  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         333
20190610 112238.208390 INFO     PET0  PASS  Obtaining access to data in Array via F90 array pointer, ESMF_ArrayDataUTest.F90, line         341
20190610 112238.208422 INFO     PET0  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTest.F90, line         354
20190610 112238.208451 INFO     PET0  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         369
20190610 112238.208497 INFO     PET0  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTest.F90, line         382
20190610 112238.208517 INFO     PET0  PASS  Destroying Array created from an allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         391
20190610 112238.208541 INFO     PET0  PASS  Creating an Array from allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         400
20190610 112238.208557 INFO     PET0  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         408
20190610 112238.208574 INFO     PET0  PASS  Obtaining access to data in Array via F90 array pointer, ESMF_ArrayDataUTest.F90, line         416
20190610 112238.208606 INFO     PET0  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTest.F90, line         429
20190610 112238.208621 INFO     PET0  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         444
20190610 112238.208652 INFO     PET0  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTest.F90, line         457
20190610 112238.208656 ERROR    PET0 ESMF_LocalArrayCreate.F90 22588 ESMF_LocalArrayF90Deallocate Unable to allocate requested memory LocalArray Deallocation
20190610 112238.208666 INFO     PET0  PASS  Destroying Array created from an allocated F90 array pointer, ESMF_ArrayDataUTest.F90, line         466
20190610 112238.208710 INFO     PET0  Number of failed tests:           0
20190610 112238.208725 INFO     PET0  Ending Test, file ESMF_ArrayDataUTest.F90, line         476
