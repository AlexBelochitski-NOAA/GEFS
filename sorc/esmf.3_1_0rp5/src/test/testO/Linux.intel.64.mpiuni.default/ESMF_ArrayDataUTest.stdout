  Beginning Test, file ESMF_ArrayDataUTest.F90, line          65
  NUMBER_OF_PROCESSORS           1
  PASS  Creating an Array from allocated F90 array pointer, ESMF_ArrayDataUTest.
 F90, line          89
  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataU
 Test.F90, line          97
  PASS  Obtaining access to data in Array via F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         105
      -12000      -12000
      -11000      -11000
      -10000      -10000
       -9000       -9000
       -8000       -8000
       -7000       -7000
       -6000       -6000
  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTes
 t.F90, line         118
  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         133
      -11943      -11943
      -10943      -10943
       -9943       -9943
       -8943       -8943
       -7943       -7943
       -6943       -6943
       -5943       -5943
  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTes
 t.F90, line         146
  PASS  Destroying Array created from an allocated F90 array pointer, ESMF_Array
 DataUTest.F90, line         155
  PASS  Creating an Array from allocated F90 array pointer using ESMF_DATA_COPY,
  ESMF_ArrayDataUTest.F90, line         164
  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         172
  PASS  Obtaining access to data in Array via F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         180
      -11943      -11943
      -10943      -10943
       -9943       -9943
       -8943       -8943
       -7943       -7943
       -6943       -6943
       -5943       -5943
  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTes
 t.F90, line         193
  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         208
      -11943      -11886
      -10943      -10886
       -9943       -9886
       -8943       -8886
       -7943       -7886
       -6943       -6886
       -5943       -5886
  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTes
 t.F90, line         221
  PASS  Destroying Array created from an allocated F90 array pointer, ESMF_Array
 DataUTest.F90, line         230
  PASS  Creating an Array from allocated F90 array pointer slice, ESMF_ArrayData
 UTest.F90, line         242
  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         250
  PASS  Obtaining access to data in Array via F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         258
      -11886      -11886
      -10886      -10886
       -9886       -9886
       -8886       -8886
       -7886       -7886
       -6886       -6886
       -5886       -5886
  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTes
 t.F90, line         271
  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         286
      -11829      -11829
      -10829      -10829
       -9829       -9829
       -8829       -8829
       -7829       -7829
       -6829       -6829
       -5829       -5829
  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTes
 t.F90, line         299
  PASS  Destroying Array created from an allocated F90 array pointer, ESMF_Array
 DataUTest.F90, line         308
  PASS  Creating an Array from allocated F90 array pointer, ESMF_ArrayDataUTest.
 F90, line         325
  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         333
  PASS  Obtaining access to data in Array via F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         341
      -12000      -12000
      -11000      -11000
      -10000      -10000
       -9000       -9000
       -8000       -8000
       -7000       -7000
       -6000       -6000
  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTes
 t.F90, line         354
  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         369
      -11943      -11943
      -10943      -10943
       -9943       -9943
       -8943       -8943
       -7943       -7943
       -6943       -6943
       -5943       -5943
  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTes
 t.F90, line         382
  PASS  Destroying Array created from an allocated F90 array pointer, ESMF_Array
 DataUTest.F90, line         391
  PASS  Creating an Array from allocated F90 array pointer, ESMF_ArrayDataUTest.
 F90, line         400
  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         408
  PASS  Obtaining access to data in Array via F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         416
      -11943      -11943
      -10943      -10943
       -9943       -9943
       -8943       -8943
       -7943       -7943
       -6943       -6943
       -5943       -5943
  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTes
 t.F90, line         429
  PASS  Printing Array created from allocated F90 array pointer, ESMF_ArrayDataU
 Test.F90, line         444
      -11943      -11886
      -10943      -10886
       -9943       -9886
       -8943       -8886
       -7943       -7886
       -6943       -6886
       -5943       -5886
  PASS  Verifying data in Array via F90 array pointer access, ESMF_ArrayDataUTes
 t.F90, line         457
  PASS  Destroying Array created from an allocated F90 array pointer, ESMF_Array
 DataUTest.F90, line         466
  Number of failed tests:           0
  Ending Test, file ESMF_ArrayDataUTest.F90, line         476
--- ESMCI::Array::print start ---
   Number of Attributes: 0
Array typekind/rank: Integer*4 / 1 
~ lower class' values ~
DistGrid dimCount = 1
deCount = 1
localDeCount = 1
~ local data in LocalArray for DE 0 ~
LocalArrayPrint: Array at address 0x01bd1ec0:
            rank = 1, kind = 3, base_addr = 0x01bd0fe0
            dim[0] = 7  
  Integer, *4, Dim 1, Data values:
( 1) =  -12000
( 2) =  -11000
( 3) =  -10000
( 4) =  -9000
( 5) =  -8000
( 6) =  -7000
( 7) =  -6000
dim 1: [0]: [1 [1 [1, 7] 7] 7]
--- ESMCI::Array::print end ---
--- ESMCI::Array::print start ---
   Number of Attributes: 0
Array typekind/rank: Integer*4 / 1 
~ lower class' values ~
DistGrid dimCount = 1
deCount = 1
localDeCount = 1
~ local data in LocalArray for DE 0 ~
LocalArrayPrint: Array at address 0x01bd1ec0:
            rank = 1, kind = 3, base_addr = 0x01bd0fe0
            dim[0] = 7  
  Integer, *4, Dim 1, Data values:
( 1) =  -11943
( 2) =  -10943
( 3) =  -9943
( 4) =  -8943
( 5) =  -7943
( 6) =  -6943
( 7) =  -5943
dim 1: [0]: [1 [1 [1, 7] 7] 7]
--- ESMCI::Array::print end ---
--- ESMCI::Array::print start ---
   Number of Attributes: 0
Array typekind/rank: Integer*4 / 1 
~ lower class' values ~
DistGrid dimCount = 1
deCount = 1
localDeCount = 1
~ local data in LocalArray for DE 0 ~
LocalArrayPrint: Array at address 0x01bd1f50:
            rank = 1, kind = 3, base_addr = 0x01bd22c0
            dim[0] = 7  
  Integer, *4, Dim 1, Data values:
( 1) =  -11943
( 2) =  -10943
( 3) =  -9943
( 4) =  -8943
( 5) =  -7943
( 6) =  -6943
( 7) =  -5943
dim 1: [0]: [1 [1 [1, 7] 7] 7]
--- ESMCI::Array::print end ---
--- ESMCI::Array::print start ---
   Number of Attributes: 0
Array typekind/rank: Integer*4 / 1 
~ lower class' values ~
DistGrid dimCount = 1
deCount = 1
localDeCount = 1
~ local data in LocalArray for DE 0 ~
LocalArrayPrint: Array at address 0x01bd1f50:
            rank = 1, kind = 3, base_addr = 0x01bd22c0
            dim[0] = 7  
  Integer, *4, Dim 1, Data values:
( 1) =  -11943
( 2) =  -10943
( 3) =  -9943
( 4) =  -8943
( 5) =  -7943
( 6) =  -6943
( 7) =  -5943
dim 1: [0]: [1 [1 [1, 7] 7] 7]
--- ESMCI::Array::print end ---
--- ESMCI::Array::print start ---
   Number of Attributes: 0
Array typekind/rank: Integer*4 / 1 
~ lower class' values ~
DistGrid dimCount = 1
deCount = 1
localDeCount = 1
~ local data in LocalArray for DE 0 ~
LocalArrayPrint: Array at address 0x01bd2300:
            rank = 1, kind = 3, base_addr = 0x01bd0fe0
            dim[0] = 7  
  Integer, *4, Dim 1, Data values:
( 1) =  -11886
( 2) =  -10886
( 3) =  -9886
( 4) =  -8886
( 5) =  -7886
( 6) =  -6886
( 7) =  -5886
dim 1: [0]: [1 [1 [1, 7] 7] 7]
--- ESMCI::Array::print end ---
--- ESMCI::Array::print start ---
   Number of Attributes: 0
Array typekind/rank: Integer*4 / 1 
~ lower class' values ~
DistGrid dimCount = 1
deCount = 1
localDeCount = 1
~ local data in LocalArray for DE 0 ~
LocalArrayPrint: Array at address 0x01bd2300:
            rank = 1, kind = 3, base_addr = 0x01bd0fe0
            dim[0] = 7  
  Integer, *4, Dim 1, Data values:
( 1) =  -11829
( 2) =  -10829
( 3) =  -9829
( 4) =  -8829
( 5) =  -7829
( 6) =  -6829
( 7) =  -5829
dim 1: [0]: [1 [1 [1, 7] 7] 7]
--- ESMCI::Array::print end ---
--- ESMCI::Array::print start ---
   Number of Attributes: 0
Array typekind/rank: Integer*4 / 1 
~ lower class' values ~
DistGrid dimCount = 1
deCount = 1
localDeCount = 1
~ local data in LocalArray for DE 0 ~
LocalArrayPrint: Array at address 0x01bd2300:
            rank = 1, kind = 3, base_addr = 0x01bd17c0
            dim[0] = 7  
  Integer, *4, Dim 1, Data values:
( 1) =  -12000
( 2) =  -11000
( 3) =  -10000
( 4) =  -9000
( 5) =  -8000
( 6) =  -7000
( 7) =  -6000
dim 1: [0]: [1 [1 [1, 7] 7] 7]
--- ESMCI::Array::print end ---
--- ESMCI::Array::print start ---
   Number of Attributes: 0
Array typekind/rank: Integer*4 / 1 
~ lower class' values ~
DistGrid dimCount = 1
deCount = 1
localDeCount = 1
~ local data in LocalArray for DE 0 ~
LocalArrayPrint: Array at address 0x01bd2300:
            rank = 1, kind = 3, base_addr = 0x01bd17c0
            dim[0] = 7  
  Integer, *4, Dim 1, Data values:
( 1) =  -11943
( 2) =  -10943
( 3) =  -9943
( 4) =  -8943
( 5) =  -7943
( 6) =  -6943
( 7) =  -5943
dim 1: [0]: [1 [1 [1, 7] 7] 7]
--- ESMCI::Array::print end ---
--- ESMCI::Array::print start ---
   Number of Attributes: 0
Array typekind/rank: Integer*4 / 1 
~ lower class' values ~
DistGrid dimCount = 1
deCount = 1
localDeCount = 1
~ local data in LocalArray for DE 0 ~
LocalArrayPrint: Array at address 0x01bd2300:
            rank = 1, kind = 3, base_addr = 0x01bd1080
            dim[0] = 7  
  Integer, *4, Dim 1, Data values:
( 1) =  -11943
( 2) =  -10943
( 3) =  -9943
( 4) =  -8943
( 5) =  -7943
( 6) =  -6943
( 7) =  -5943
dim 1: [0]: [1 [1 [1, 7] 7] 7]
--- ESMCI::Array::print end ---
--- ESMCI::Array::print start ---
   Number of Attributes: 0
Array typekind/rank: Integer*4 / 1 
~ lower class' values ~
DistGrid dimCount = 1
deCount = 1
localDeCount = 1
~ local data in LocalArray for DE 0 ~
LocalArrayPrint: Array at address 0x01bd2300:
            rank = 1, kind = 3, base_addr = 0x01bd1080
            dim[0] = 7  
  Integer, *4, Dim 1, Data values:
( 1) =  -11943
( 2) =  -10943
( 3) =  -9943
( 4) =  -8943
( 5) =  -7943
( 6) =  -6943
( 7) =  -5943
dim 1: [0]: [1 [1 [1, 7] 7] 7]
--- ESMCI::Array::print end ---
